huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -1039,6 +1100,19 @@ func updateRdsInstanceName(d *schema.ResourceData, client *golangsdk.ServiceClie
 	return nil
 }
 
+func updateRdsInstanceDescription(client *golangsdk.ServiceClient, instanceID, description string) error {
+	modifyAliasOpts := instances.ModifyAliasOpts{
+		Alias: description,
+	}
+	log.Printf("[DEBUG] Modify RDS instance description opts: %+v", modifyAliasOpts)
+	r := instances.ModifyAlias(client, modifyAliasOpts, instanceID)
+	if r.Result.Err != nil {
```

Comment:
use r.Err will be fine

```suggestion
	if r.Err != nil {
```

Reply:
done